docs/src/inputs/prompt/examples/exampleSuccessPrompt.jsx

```diff
@@ -0,0 +1,48 @@
+import {
+    Button,
+    Icon,
+    Prompt,
+} from 'react-cm-ui';
+import React, { useState } from 'react';
+
+function ExampleSuccessPrompt() {
+    const [showPrompt, setShowPrompt] = useState(false);
+
+    function onPromptClick() {
+        setShowPrompt(true);
+    }
+
+    function onYesClick() {
+        setShowPrompt(false);
+    }
+
+    function onNoClick() {
+        setShowPrompt(false);
+    }
+
+    return (
+        <div>
+            <Prompt
```

Comment:
This example needs all three states show as in:

![image](https://user-images.githubusercontent.com/1100790/98309073-6b366780-1f7e-11eb-97f5-440f7272684a.png)

Reply:
Fixed